spec/requests/adjustments_requests_spec.rb

```diff
@@ -72,10 +72,63 @@
 
       context "csv" do
         let(:response_format) { 'csv' }
+        let(:storage_location) { create(:storage_location, organization: organization) }
+        let(:item1) { create(:item, name: "Item One", organization: organization) }
+        let(:item2) { create(:item, name: "Item Two", organization: organization) }
+
+        let!(:adjustment1) do
+          adj = create(:adjustment,
+            organization: organization,
+            storage_location: storage_location,
+            comment: "First adjustment",
+            created_at: 1.day.ago)
+          adj.line_items << build(:line_item, quantity: 10, item: item1, itemizable: adj)
+          adj.line_items << build(:line_item, quantity: 5, item: item2, itemizable: adj)
+          adj
+        end
+
+        let!(:adjustment2) do
+          adj = create(:adjustment,
+            organization: organization,
+            storage_location: storage_location,
+            comment: "Second adjustment",
+            created_at: 5.days.ago)
+          adj.line_items << build(:line_item, quantity: -5, item: item1, itemizable: adj)
+          adj
+        end
 
-        before { adjustment }
+        before { get adjustments_path(format: 'csv') }
 
-        it { is_expected.to be_successful }
+        it "returns a CSV file" do
+          expect(response).to be_successful
+          expect(response.header['Content-Type']).to include 'text/csv'
+        end
+
+        it "includes appropriate headers and data" do
+          csv = <<~CSV
+            Created date,Storage Area,Comment,# of changes,#{item1.name},#{item2.name}
```

Comment:
We know the name, can we put the hardcoded value in here?

Reply:
good point, let me quickly change that :)

Reply:
@dorner should I squash this and the last change to a single commit and then force push? to not create too much commit noise

Reply:
@cyang-el no, that's what merge commits are for. :smile: Force push causes havoc with existing comments.